### ISSUE TITLE
v2.2.0 causes webpack to run longer times

### ISSUE THREAD

**Issue by yoyo837** [webpack-contrib/stylelint-webpack-plugin#234]
- Operating System: macOS 11.4
- Node Version: 14.17.1
- NPM Version: 6.14.13
- webpack Version: 4.46.0
- stylelint-webpack-plugin Version: 2.2.0

### Expected Behavior

`exclude: 'node_modules',` by default works well，

### Actual Behavior

`exclude: 'node_modules',` by default not works，It causes `webpack` to run longer.

### Code

```js
      new StylelintPlugin({
        files: '**/*.s(a|c)ss',
      })
```

### How Do We Reproduce?

Just upgrade `stylelint-webpack-plugin` from 2.1.1 to 2.2.0.

2.1.1 is works very well.

It may also be caused by other reasons, this is just my personal guess, because it seems looks for more files in more folders.

So i guess the `folders` is `node_modules`?

**Comment by BenXueYou** [comment]
the same as me

**Comment by 0x1af2aec8f957** [comment]
The same problem

**Comment by ricardogobbosouza** [comment]
Hi @yoyo837 
Please provide a repository to reproduce

**Comment by yoyo837** [comment]
> Hi @yoyo837
> Please provide a repository to reproduce

thank you for your reply. I will try it, but maybe not these days.

**Comment by msarakon** [comment]
Same here, I'm running this plugin on Storybook webpack config and `v2.1.1` works fine but `v2.2.0` makes the build super slow.

- OS: macOS 10.15.7
- Node: 14.16.1
- NPM: 6.14.8
- Webpack: 5.37.1

**Comment by tmcconechy** [comment]
If you need a repo to reproduce this ours is public. Steps:

1. clone https://github.com/infor-design/enterprise-wc
2. run `npm run build` -> takes about 9564
3. update stylelint-webpack-plugin in the package.json to 2.2.2
2. run `npm run build` -> takes around  158505 ms

**Comment by ricardogobbosouza** [comment]
Thanks @tmcconechy 
I will check as soon as possible

**Comment by yoyo837** [comment]
> If you need a repo to reproduce this ours is public. Steps:
> 
> 1. clone https://github.com/infor-design/enterprise-wc
> 2. `npm install` and run `npm run build` -> takes about 9564ms
> 3. update stylelint-webpack-plugin in the package.json to 2.2.0
> 4. run `npm run build` -> takes around  158505 ms
> 
> Is it not ignoring node_modules by default anymore? Do we need a setting?

Thanks for the quick reproduce.
I tried to add that setting, but it still doesn't work.

**Comment by ricardogobbosouza** [comment]
Fixed [v2.2.1](https://github.com/webpack-contrib/stylelint-webpack-plugin/releases/tag/v2.2.1)

**Comment by ricardogobbosouza** [comment]
Thanks @yoyo837 @tmcconechy
